src/transformStateWithClones.js

```diff
@@ -66,6 +66,38 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
-}
+  const newStates = [];
+  const newState = {
+    ...state,
+  };
+
+  for (const transform of transforms) {
```

Comment:
You can do desctructuring
for (const [operation, poperties] of transforms)
and then will be less repeated code